sdk/automation/Azure.ResourceManager.Automation/tests/AutomationManagementTestBase.cs

```diff
@@ -25,17 +28,37 @@ protected AutomationManagementTestBase(bool isAsync)
         }
 
         [SetUp]
-        public void CreateCommonClient()
+        public async Task CreateCommonClient()
         {
             Client = GetArmClient();
+            DefaultSubscription = await Client.GetDefaultSubscriptionAsync().ConfigureAwait(false);
         }
 
-        protected async Task<ResourceGroupResource> CreateResourceGroup(SubscriptionResource subscription, string rgNamePrefix, AzureLocation location)
+        protected async Task<ResourceGroupResource> CreateResourceGroupAsync()
         {
-            string rgName = Recording.GenerateAssetName(rgNamePrefix);
-            ResourceGroupData input = new ResourceGroupData(location);
-            var lro = await subscription.GetResourceGroups().CreateOrUpdateAsync(WaitUntil.Completed, rgName, input);
-            return lro.Value;
+            var resourceGroupName = Recording.GenerateAssetName("testRG-");
+            var rgOp = await DefaultSubscription.GetResourceGroups().CreateOrUpdateAsync(
+                WaitUntil.Completed,
+                resourceGroupName,
+                new ResourceGroupData(DefaultLocation)
+                {
+                    Tags =
+                    {
+                        { "test", "env" }
+                    }
+                });
+            if (Mode == RecordedTestMode.Playback)
```

Comment:
Please refer the WebPubSub case in https://github.com/Azure/azure-sdk-for-net/pull/35831 which not exclude recording for Network related operations but filter out the version of Network APIs

Reply:
Change after that  PR got merged

Reply:
The pr was merged

Reply:
fixed